src/main/java/io/cdap/plugin/servicenow/sink/output/ServiceNowOutputFormatProvider.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.cdap.plugin.servicenow.sink.output;
+
+import io.cdap.cdap.api.data.batch.OutputFormatProvider;
+import org.apache.hadoop.conf.Configuration;
+
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.stream.StreamSupport;
+
+/**
+ * ServiceNow Output Format Provider Class
+ */
```

Comment:
Please add a valid javaDoc for this class.

Reply:
Please suggest a valid JavaDoc for this class.

Reply:
Provides {@link ServiceNowOutputFormat} to output values for tables. (Just a suggestion)

Reply:
Added 843ee483f38d0090b76f1a8be5e6bcf2a967bd1d